CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [#11828](https://github.com/CocoaPods/CocoaPods/pull/11828)
   [#11808](https://github.com/CocoaPods/CocoaPods/issues/11808)
 
+# Xcode 14.3 fix: `pod lib lint` warning generation from main.m.
```

Comment:
nit: two spaces after the end of this line for markdown formatting

Reply:
Oops, I guess it's been awhile. 😄  Done. Thanks!